api/api/src/main/scala/uk/ac/wellcome/platform/api/services/RelatedWorkService.scala

```diff
@@ -31,7 +24,7 @@ class RelatedWorkService(elasticsearchService: ElasticsearchService)(
   def retrieveRelatedWorks(index: Index,
                            work: IdentifiedWork): Future[Result[RelatedWorks]] =
     work.data.collectionPath match {
-      case None => Future.successful(Right(RelatedWorks(Nil, Nil, Nil, Nil)))
+      case None => Future.successful(Right(RelatedWorks(Some(Nil), Some(Nil), Some(Nil), Some(Nil))))
```

Comment:
Could be worth adding a `def empty` for this case to the companion object as it's used in a couple of places / looks nice

Reply:
There is already a `def none` for when its `None`, this is `Some(Nil)` :see_no_evil: .. This means there actually no related works, the other means we don't actually know at that point, maybe could be `unknown` / `none` or something?

Reply:
Yeah the naming is tricky... how about `RelatedWorks.nil`?

Reply:
:+1: I like `RelatedWorks.nil` and `RelatedWorks.unknown`